### ISSUE TITLE
Bookmarks uploaded from CSV file

### ISSUE THREAD

**Issue by paggps** [fin10/GearBrowser#4]
GearBrowser is an excellent app for the Gear smartwatches.
Then only problem I have is how to backup/restore the list of Bookmarks: in smartwatches it is very common that you have to "reset" or "unlink" the watch from the companion phone, and that means starting all over again. The bookmark configuration of Gear Browser is not saved in any place we (the users) can access, and URLs sometimes are VERY LONG and difficult to type on the Gear screen.

So my suggestion to solve this is as follows:

Allow GearBrowser to read (only read) an external "bookmark file", which may be located on a specific place on the phone, or on the GoogleDrive account related to the Google account in use.
So GearBrowser can access a simple CSV (comma separated values) file in a format like this:
BOOKMARK NAME, URL

Example
MyURL1,http://www.myhostrev.com:8087/cgi-bin/CGIProxy.fcgi?cmd=Picture&usr=admin&mypwd
MyURL2,http://www.mynewhost.com:2527/cgi-bin/CGIProxy.fcgi?cmd=Picture&usr=admin&mypwd

Thanks

**Comment by Changyong-Jeon** [comment]
Good idea! I will try to find a solution to support it. But unfortunately Google Drive will not be a solution because login process will be a terrible experience to user on tiny screen.

**Comment by fin10** [comment]
Good idea! I will try to find a solution to support it. But unfortunately Google Drive will not be a solution because login process will be a terrible experience to user on tiny screen.
